lib/config.js

```diff
@@ -143,6 +142,10 @@ Config.prototype.buildArgs = function () {
     }
   }
 
+  if (this.targetOS === 'android') {
```

Comment:
there is already a section for `android` below https://github.com/brave/brave-browser/pull/6839/files#diff-f7bf2b665273dfa66ffa4ad7c0a52bb9R185

Reply:
Fixed.